docs/docs/developers/uploadTrackMetadata.md

```diff
@@ -101,7 +103,7 @@ const { trackId } = await audiusSdk.tracks.uploadTrack({
 
 ## Specifying Stream Conditions
 
-Use the `StreamConditions` field to specify the criteria required to unlock a track.
```

Comment:
`streamConditions` is the field, right? `AccessConditions` is the type of the field?

Reply:
yes sir. will update

Reply:
just remembered this is fixed in upcoming PR